strato

```diff
@@ -113,6 +114,13 @@ function keygen {
   docker run --rm --entrypoint=keygen ${STRATO_IMAGE:-$DC_STRATO_IMAGE} --count="$1"
 }
 
+function mountStratoVolume {
+  echo -e "${BYellow}STRATO container already exists. Restarting"
+  docker restart strato_strato_1
```

Comment:
With `restart`, the same container is used, which is why the logs had the same timestamps in them. What we actually want to do is start a new container (perhaps with a different image), and keep the same volume

Reply:
I want to test further whether `STRATO_IMAGE=xxx docker-compose up` depends on whether the docker-compose has been modified. If it has, maybe we reject the --upgrade-strato. If it hasn't, we run the `up`.

We also talked about creating kafka and redis volumes, that might be a safer way make all containers restartable

Reply:
Done. I was to change the `--remove` usage to remove the redis, strato, and kafka volumes, then I found that `remove` wants to `Gently stopping and removing STRATO containers, keeping volumes${NC}"`, which looks like we want to keep their volumes after `remove`

Reply:
The text of the warning or maybe the name of the can be changed, but the semantics of --remove are that user data is kept and chain data is wiped.